addons/hr_timesheet/controllers/portal.py

```diff
@@ -26,11 +25,11 @@ def _prepare_home_portal_values(self, counters):
 
     def _get_searchbar_inputs(self):
         return {
-            'all': {'input': 'all', 'label': _('Search in All')},
-            'employee': {'input': 'employee', 'label': _('Search in Employee')},
-            'project': {'input': 'project', 'label': _('Search in Project')},
-            'task': {'input': 'task', 'label': _('Search in Task')},
-            'name': {'input': 'name', 'label': _('Search in Description')},
+            'name': {'input': 'name', 'label': _('Search in Description'), 'sequence': 10},
+            'employee_id': {'input': 'employee_id', 'label': _('Search in Employee'), 'sequence': 20},
+            'project_id': {'input': 'project_id', 'label': _('Search in Project'), 'sequence': 30},
+            'task_id': {'input': 'task_id', 'label': _('Search in Task'), 'sequence': 40},
+            'parent_task_id': {'input': 'parent_task_id', 'label': _('Search in Parent Task'), 'sequence': 70},
```

Comment:
I am wondering if it should not be better to use `_lt` function instead of `_` function. 🤔